### DISCUSSION TITLE
[all products] Windows Support Documentation

### DISCUSSION THREAD

**Discussion by YazeedAlKhalaf** [firebase/flutterfire#5795] [Feature request] [answered]
This issue is to document the process of adding Windows support to FlutterFire plugins and to discuss the possibility.

**Comment by esDotDev** [comment]
It would be great to know even some rough timeline on when Windows development will begin, and whether it's considered fairly feasible / low effort.

**Comment by Ehesp** [comment]
It's considered, however priority right now is Android, iOS & Web.

**Comment by YazeedAlKhalaf** [comment]
So by changing the title to documentation, do you mean that the implementation is there but needs docs, or am I getting it wrong?

**Comment by jeffmilanes** [comment]
any updates?

**Comment by WashingtonlGarcia** [comment]
> alguma atualização?


I also want to know

**Comment by DevKang** [comment]
I am waiting for it ~! :)

**Comment by Ehesp** [comment]
I think it's worth dropping this issue as relevant: https://github.com/firebase/firebase-cpp-sdk/issues/42#issuecomment-591073497

The C++/Windows SDK is designed for mobile games, thus a lot of the implementations are stubs - we'd end up with a fragmented experience.

**Reply by cbenhagen** [reply]
@Ehesp The linked information might be outdated. The C++ SDK is now in beta and while some of the implementations are still stubs, the most important parts are not: https://firebase.google.com/docs/cpp/setup#libraries-desktop

@morganchen12 would you still not recommend using the C++ SDK for Desktop apps?

**Reply by Ehesp** [reply]
@cbenhagen it says on that link:

> Caution: Firebase C++ SDK desktop support is a beta feature. This feature is intended only for workflows during the development of your game, not for publicly shipping code.

**Reply by morganchen12** [reply]
If your use of Firebase does not extend to any of the remaining unsupported libraries, you absolutely can use the C++ SDKs on desktop, but as @Ehesp mentioned do keep in mind that the C++ SDKs on desktop are still in beta.

**Comment by YazeedAlKhalaf** [comment]
Any updates on the matter?

**Comment by JohnGalt1717** [comment]
This becomes a blocker pretty quickly. Same for linux support.

**Comment by Salakar** [comment, marked as answer]
Duplicate of #5557
